src/domains/guest/screens/Playground.tsx

```diff
@@ -0,0 +1,103 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import { isWeekend } from "@internationalized/date";
```

Comment:
`@internationalized/date` is a library that pairs with `react-aria-components` and provides date comparing functions. We currently use `date-fns` for this, so maybe we should look into integrating it instead.